scraper/src/strategies/default_strategy.py

```diff
@@ -233,7 +246,7 @@ def _get_selector_all(levels, selectors):
         for level in levels:
             level_selector = selectors[level]
 
-            if len(level_selector['selector']) > 0:
+            if len(level_selector['selector']) > 0 and not level_selector['global']:
```

Comment:
short-circuit, better the other way around (nested method)

Reply:
👍